app/code/Magento/Cms/Test/Unit/Controller/Adminhtml/Wysiwyg/DirectiveTest.php

```diff
@@ -106,6 +106,9 @@ class DirectiveTest extends TestCase
      */
     private $driverMock;
 
+    /**
+     * @inheirtDoc
```

Comment:
Typo here and in all the `setUp` methods below. Also, I don't think that we should camelcase it, based on the [phpdoc manual](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlineinheritdoc.pkg.html).
```suggestion
     * @inheritdoc
```